osm_cli/cli/main.py

```diff
@@ -13,6 +14,30 @@ def osm():
 @click.argument("file_path", type=click.Path(exists=True))
 @click.argument("file_id", type=str)
 def pdf_xml(file_path, file_id):
-    """This function converts a file from PDF to XML"""
-    # Function Implementation
+    """
+    Args:
+        file_path (file path): First parameter
+        file_id (string): Second parameter
+    Returns:
+        Creates an XML file in the directory xmls_sciencebeam
+    """
+    try:
+        converter = PDFConverter()
+
+        if not converter.is_docker_running():
+            raise click.ClickException('Please make sure the docker is running')
+
+        if not converter.is_host_ready():
+            raise click.ClickException('The converter server is offline')
+
+        xml_content = converter.convert(file_path)
+        # Save the converted xml contents
+        output_file: str = f'docs/examples/sciencebeam_xml_outputs/{file_id}.xml'
```

Comment:
remove this. the docs directory shouldn't be written to by the code. the user has provided an outputfile path, so just use that. You could consider adding logic that the output directory exists but i suspect the default error when it fails to write will be interpretable for now.

Reply:
We have removed the hardcoded output path for the docs directory and used outputfile path as suggested.